tigera-operator.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: tigera-operator
+  version: 1.33.0
+  epoch: 0
+  description: Kubernetes operator for installing Calico and Calico Enterprise
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - tigera-operator=${{package.full-version}}
```

Comment:
If a package provides itself, it will cause a conflict with `apk`.

Reply:
Thanks, that was the issue